### PR TITLE
#29 - Create database deploy action

### PR DESCRIPTION
**Pull request by JustasBelevicius**

### CODE REVIEW

File: .github/workflows/nuke_database.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+DO $$
```
Comment (JustasBelevicius) [inline .github/workflows/nuke_database.sql:1]: We wouldn't need this if our SQL files contained `DROP IF EXISTS` logic before the creation.

File: .github/workflows/deploy-database.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Nuke & Redeploy Database (not prod safe)
```
Comment (ignasposka) [inline .github/workflows/deploy-database.yml:1]: Can we make it prod safe initially?

**Review by ignasposka — APPROVED** [review]

File: .github/workflows/deploy-database.yml (hunk 2 above)
Comment (AzisK) [reply .github/workflows/deploy-database.yml:1]: I believe @JustasBelevicius has a plan for it to create a separate folder just updating and not nuking in the future. For development stage until MVP, I believe we are good to go with this

**Review by AzisK — APPROVED** [review]
Looks alrighty

**Pull request merged**